Pester.psm1

```diff
@@ -1077,6 +1077,7 @@ function Invoke-Pester {
 
         $script:mockTable = @{}
         Remove-MockFunctionsAndAliases
+        Set-ConflictingParameterNames
```

Comment:
@FISHMANPET Having this here means that it will get executed every time tests are run, which is what we are trying to avoid.

Just move the function invocation as the last line before the export-modulemember section in Pester.psm1 (line 1531)